Description: Turn off manpages generation
 Turn off manpages generation to work around FTBFS with broken doxygen.
 I will apply it upstream myself soon.
Author: Theppitak Karoonboonyanan <thep@debian.org>
Bug-Debian: http://bugs.debian.org/573663
Last-Update: 2010-03-14

* doc/Doxyfile.in: Turn GENERATE_MAN off.
* doc/Makefile.am: Remove man targets.

Index: libthai/doc/Doxyfile.in
===================================================================
--- libthai.orig/doc/Doxyfile.in	2010-03-14 16:57:56.000000000 +0700
+++ libthai/doc/Doxyfile.in	2010-03-14 16:58:09.000000000 +0700
@@ -205,7 +205,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the man page output
 #---------------------------------------------------------------------------
-GENERATE_MAN           = YES
+GENERATE_MAN           = NO
 MAN_OUTPUT             = man
 MAN_EXTENSION          = .3
 MAN_LINKS              = YES
Index: libthai/doc/Makefile.am
===================================================================
--- libthai.orig/doc/Makefile.am	2010-03-14 16:57:38.000000000 +0700
+++ libthai/doc/Makefile.am	2010-03-14 16:57:44.000000000 +0700
@@ -1,92 +1,5 @@
 if ENABLE_DOXYGEN_DOC
 
-man_MANS = \
-	$(top_builddir)/doc/man/man3/thailib.h.3	\
-	$(top_builddir)/doc/man/man3/th_brk.3	\
-	$(top_builddir)/doc/man/man3/thbrk.h.3	\
-	$(top_builddir)/doc/man/man3/th_brk_line.3	\
-	$(top_builddir)/doc/man/man3/thcell.h.3	\
-	$(top_builddir)/doc/man/man3/th_init_cell.3	\
-	$(top_builddir)/doc/man/man3/th_next_cell.3	\
-	$(top_builddir)/doc/man/man3/th_prev_cell.3	\
-	$(top_builddir)/doc/man/man3/th_make_cells.3	\
-	$(top_builddir)/doc/man/man3/thcoll.h.3	\
-	$(top_builddir)/doc/man/man3/th_strcoll.3	\
-	$(top_builddir)/doc/man/man3/th_strxfrm.3	\
-	$(top_builddir)/doc/man/man3/thctype.h.3	\
-	$(top_builddir)/doc/man/man3/th_istis.3	\
-	$(top_builddir)/doc/man/man3/th_isthai.3	\
-	$(top_builddir)/doc/man/man3/th_iseng.3	\
-	$(top_builddir)/doc/man/man3/th_isthcons.3	\
-	$(top_builddir)/doc/man/man3/th_isthvowel.3	\
-	$(top_builddir)/doc/man/man3/th_isthtone.3	\
-	$(top_builddir)/doc/man/man3/th_isthdiac.3	\
-	$(top_builddir)/doc/man/man3/th_isthdigit.3	\
-	$(top_builddir)/doc/man/man3/th_isthpunct.3	\
-	$(top_builddir)/doc/man/man3/th_istaillesscons.3	\
-	$(top_builddir)/doc/man/man3/th_isovershootcons.3	\
-	$(top_builddir)/doc/man/man3/th_isundershootcons.3	\
-	$(top_builddir)/doc/man/man3/th_isundersplitcons.3	\
-	$(top_builddir)/doc/man/man3/th_isldvowel.3	\
-	$(top_builddir)/doc/man/man3/th_isflvowel.3	\
-	$(top_builddir)/doc/man/man3/th_isupvowel.3	\
-	$(top_builddir)/doc/man/man3/th_isblvowel.3	\
-	$(top_builddir)/doc/man/man3/th_chlevel.3	\
-	$(top_builddir)/doc/man/man3/th_iscombchar.3	\
-	$(top_builddir)/doc/man/man3/thinp.h.3	\
-	$(top_builddir)/doc/man/man3/th_isaccept.3	\
-	$(top_builddir)/doc/man/man3/th_validate.3	\
-	$(top_builddir)/doc/man/man3/thrend.h.3	\
-	$(top_builddir)/doc/man/man3/th_render_cell_tis.3	\
-	$(top_builddir)/doc/man/man3/th_render_cell_win.3	\
-	$(top_builddir)/doc/man/man3/th_render_cell_mac.3	\
-	$(top_builddir)/doc/man/man3/th_render_text_tis.3	\
-	$(top_builddir)/doc/man/man3/th_render_text_win.3	\
-	$(top_builddir)/doc/man/man3/th_render_text_mac.3	\
-	$(top_builddir)/doc/man/man3/thstr.h.3	\
-	$(top_builddir)/doc/man/man3/th_normalize.3	\
-	$(top_builddir)/doc/man/man3/thwbrk.h.3	\
-	$(top_builddir)/doc/man/man3/th_wbrk.3	\
-	$(top_builddir)/doc/man/man3/th_wbrk_line.3	\
-	$(top_builddir)/doc/man/man3/thwchar.h.3	\
-	$(top_builddir)/doc/man/man3/th_tis2uni.3	\
-	$(top_builddir)/doc/man/man3/th_tis2uni_line.3	\
-	$(top_builddir)/doc/man/man3/th_winthai2uni.3	\
-	$(top_builddir)/doc/man/man3/th_macthai2uni.3	\
-	$(top_builddir)/doc/man/man3/th_uni2tis.3	\
-	$(top_builddir)/doc/man/man3/th_uni2tis_line.3	\
-	$(top_builddir)/doc/man/man3/th_uni2winthai.3	\
-	$(top_builddir)/doc/man/man3/th_uni2macthai.3	\
-	$(top_builddir)/doc/man/man3/thwcoll.h.3	\
-	$(top_builddir)/doc/man/man3/th_wcstrcoll.3	\
-	$(top_builddir)/doc/man/man3/th_wcstrxfrm.3	\
-	$(top_builddir)/doc/man/man3/thwctype.h.3	\
-	$(top_builddir)/doc/man/man3/th_wcistis.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthai.3	\
-	$(top_builddir)/doc/man/man3/th_wciseng.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthcons.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthvowel.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthtone.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthdiac.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthdigit.3	\
-	$(top_builddir)/doc/man/man3/th_wcisthpunct.3	\
-	$(top_builddir)/doc/man/man3/th_wcistaillesscons.3	\
-	$(top_builddir)/doc/man/man3/th_wcisovershootcons.3	\
-	$(top_builddir)/doc/man/man3/th_wcisundershootcons.3	\
-	$(top_builddir)/doc/man/man3/th_wcisundersplitcons.3	\
-	$(top_builddir)/doc/man/man3/th_wcisldvowel.3	\
-	$(top_builddir)/doc/man/man3/th_wcisflvowel.3	\
-	$(top_builddir)/doc/man/man3/th_wcisupvowel.3	\
-	$(top_builddir)/doc/man/man3/th_wcisblvowel.3	\
-	$(top_builddir)/doc/man/man3/th_wcchlevel.3	\
-	$(top_builddir)/doc/man/man3/thwstr.h.3	\
-	$(top_builddir)/doc/man/man3/th_wnormalize.3	\
-	$(top_builddir)/doc/man/man3/wtt.h.3	\
-	$(top_builddir)/doc/man/man3/TACchtype.3	\
-	$(top_builddir)/doc/man/man3/TACio_op.3
-
-$(man_MANS): doxygen.stamp
-
 all-local: doxygen.stamp
 
 doxygen.stamp: $(top_srcdir)/include/thai/*.h $(top_srcdir)/src/*/*.c
